vast/integration/reference/server-zeek-multiple-imports/step_02.ref

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "zeek.conn": {
+      "record": {
+        "ts": {
+          "timestamp": "time"
+        },
+        "uid": {
+          "type": "string",
+          "attributes": [
+            {
+              "index": "hash"
+            }
+          ]
```

Comment:
Why is attributes a list of objects instead of just an object? Afaik this is inconsistent with the modules proposal.

Reply:
It is consistent with the proposal, but I agree that a record makes a lot more sense here.

Reply:
We talked about this and agreed to make the change.

Reply:
Done.